src/locales/fi.json

```diff
@@ -320,7 +320,9 @@
       "phone": "Puhelinnumero",
       "email": "Sähköpostiosoite",
       "comment": "Huomiot",
-      "selectInvoicingTypePaperInvoice": "Laskutus paperisena"
+      "selectInvoicingTypePaperInvoice": "Laskutus paperisena",
+      "stickerNumberAndSeason": "Tarra ja kausi",
+      "stickerNumberAndSeasonPlaceholder": "Numero vuosi/vuosi"
```

Comment:
vuosi/vuosi? 🤔

Reply:
This might need some discussion still:) Seasons are in the form "2020/2021" So it's "start year / end year", but it should fit in the input box nicely.

Reply:
OK, now they're "Nro alkuvuosi/loppuvuosi" which is more clear. Seems the input box doesn't _need_ top be 25% width, so the box will stretch to fit whole string now.